app/uk/gov/hmrc/membersprotectionsenhancements/connectors/NpsConnector.scala

```diff
@@ -52,25 +50,54 @@ class NpsConnector @Inject() (val config: AppConfig, val http: HttpClientV2) ext
     s"Basic $encoded"
   }
 
+  private def checkIdsMatch(
+    requestCorrelationId: CorrelationId,
+    responseCorrelationId: CorrelationId,
+    extraLoggingContext: Option[String]
+  ): CorrelationId =
+    if (requestCorrelationId.value == responseCorrelationId.value) {
+      responseCorrelationId
```

Comment:
Is it possible to add a unit test for this scenario? `if (requestCorrelationId.value == responseCorrelationId.value)`

Reply:
I prefer this version, yes